arrow/arrow.py

```diff
@@ -962,6 +972,46 @@ def humanize(
                 if trunc(abs(delta)) != 1:
                     granularity += "s"
                 return locale.describe(granularity, delta, only_distance=only_distance)
+
+            else:
+                timeframes = []
+                if "year" in granularity:
```

Comment:
Just a minor style comment, but putting spaces between the singular if statements may be cleaner since they are not chained conditionals (e.g. `if...elif...elif...else`).